python/existing-vpc-new-ec2-ebs-userdata/app.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+
+from cdk_vpc_ec2.cdk_vpc_ec2_stack import CdkVpcEc2Stack
+
+env_cn = core.Environment(account="1111111111", region="cn-northwest-1")
```

Comment:
Leave inline comment indicating this needs to be replaced with personal account details